example/bayesian-methods/bdk_demo.py

```diff
@@ -58,6 +62,9 @@ def backward(self, out_grad, in_data, out_data, in_grad):
 
 
 class LogSoftmax(mx.operator.NumpyOp):
+    """
+    Generate helper functions to evaluate softmax loss function
```

Comment:
loss function? Shouldn't this be logarithm of softmax

Reply:
nitpick : At times you have used "calculate" and at times "Generate helper functions to evaluate"
Should have consistency right?

Reply:
Thanks for pointing out the misdescription of softmax, and I agree to use "calculate" here, it's a better description.